admin_guide/diagnostics_tool.adoc

```diff
@@ -116,37 +116,48 @@ current master) should be able to diagnose the status of infrastructure such as
 nodes, registry, and router. In each case, running `oc adm diagnostics` looks
 for the client configuration in its standard location and uses it if available.
 
-[[additional-cluster-health-checks]]
-== Additional Diagnostic Checks via Ansible
-
-// TODO: add link to OCP image once it is available
-
-Some additional diagnostic checks are available through the *openshift-ansible*
-container image. See the image's link:https://github.com/openshift/openshift-ansible/blob/master/README_CONTAINER_IMAGE.md[source repository] for usage information.
-
-The following health checks belong to a diagnostic task meant to be run against
-the Ansible inventory file for a deployed {product-title} cluster. They can
-report common problems for the current {product-title} installation.
+[[ansible-based-tooling-health-checks]]
+== Ansible-based Health Checks
+
+// tag::ansible-based-health-checks-intro[]
+Additional diagnostic health checks are available through the
+xref:../install_config/install/advanced_install.adoc#install-config-install-advanced-install[Ansible-based tooling] used to install and manage {product-title} clusters. They can report
+common deployment problems for the current {product-title} installation.
```

Comment:
I consider this a bug in the check runner :cry: 
It is not the user's fault the implications of how the checks are implemented. We should fix it, since IIUC it did not start that way.